include/ddsrouter/communication/PayloadPool.hpp

```diff
@@ -158,6 +158,13 @@ class PayloadPool : public fastrtps::rtps::IPayloadPool
 
 protected:
 
+    bool reserve_(
```

Comment:
Returned bool is used nowhere, make it void or make use of it.

Reply:
Actually reserve could fail, but `reserve` method does not return return value in Fast DDS, so I would say the issue is there and not here.
I understand it is useless, but I would leave it this way. It is better to have it useless than change the API in the future.